src/pages/login/Login.scss

```
@@ -0,0 +1,170 @@
+.haimLogin {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  overflow-y: scroll;
+  z-index: 99999;
+  padding-bottom: 10%;
+  .wrapLogin {
+    width: 45%;
+    margin: 0 auto;
+    .loginHeader {
+      .backBtn {
+        margin-top: 5%;
```

```suggestion
  z-index: 99999;
  padding-bottom: 10%;
  
  .wrapLogin {
    width: 45%;
    margin: 0 auto;
    
    .loginHeader {
      .backBtn {
        margin-top: 5%;
```
선택자 사이의 가독성을 위해서 한칸씩 띄워주세요!
다른 scss 파일에도 동일하게 적용해 주세요!

---

띄워쓰기 해도 되는거였군요! 띄워쓰니 정말 가독성이 올라가는게 보입니다!! 오오